maths/prime_check.py

```diff
@@ -9,6 +9,18 @@ def prime_check(number):
     Check to See if a Number is Prime.
 
     A number is prime if it has exactly two dividers: 1 and itself.
+    >>>prime_check(2)
```

Comment:
There needs to be one space between >>> and prime_check(2).

Reply:
As discussed in CONTRIBUTING.md, please run these tests locally with `python3 -m doctest -v maths/prime_check.py`